streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -742,13 +742,19 @@ public boolean isProcessable(final long wallClockTime) {
         if (hasPendingTxCommit) {
             // if the task has a pending TX commit, we should just retry the commit but not process any records
             // thus, the task is not processable, even if there is available data in the record queue
+            if (log.isDebugEnabled()) {
```

Comment:
What is the reason for calling `isDebugEnabled()` only at this specific location and not in the former debug level loggings?